readthedocs/projects/querysets.py

```diff
@@ -148,6 +147,18 @@ def dashboard(self, user):
     def api(self, user=None):
         return self.public(user)
 
+    def only_owner(self, user):
+        """
+        Returns projects where `user` is the only owner.
+
+        Projects that belong to organizations aren't included.
+        """
```

Comment:
Reading the comment here I think it's clear. What about naming this method "solo_owner" instead? I think it's easier to read and will avoid the confusion that I had in the previous comment. cc @ericholscher 

> Note this also applies for the method in the organization manager

Reply:
`solo` doesn't seem more clear to me than `only`. We can probably be more verbose if needed.. `has_single_owner`?

Reply:
I wouldn't prepend `is_` to it because I'd expect a Yes/No (boolean) answer in that case. I think `Project.objects.single_owner` works better for me than `only_onwer` 👍🏼

Reply:
`single_owner` seems good 👍